lib/suse/connect/zypper.rb

```diff
@@ -20,7 +20,7 @@ class << self
         # Returns an array of all installed products, in which every product is
         # presented as a hash.
         def installed_products
-          zypper_out = call('--xmlout --non-interactive products -i', false)
+          zypper_out = call('--no-refresh --xmlout --non-interactive products -i', false)
```

Comment:
It should be a part of integration testing were we calling --status on deleted system. I will check it again